pom.xml

```diff
@@ -1733,7 +1733,7 @@
               <configuration>
                 <rules>
                   <requireMavenVersion>
-                    <version>${maven.version}</version>
+                    <version>[3.3,)</version>
```

Comment:
This should just be `[${maven.version},)` Because this minimum version is provided by the build it's OK to require it.